Vizzuality/heco-invest
# Change style of discover pages on mobile version

This PR updates the mobile version of the discover pages

## Testing instructions

This task is not about the search functionality nor the filters. It’s about how the page looks like when it’s loaded:

- The header’s background goes until the tabs.
- The tabs are different.
- The search bar is less tall.
- The sorting and ordering options are smaller.
- The cards have a smaller font size.
- The grant and deadline are aligned horizontally (open call cards).

More: see [HeCo - Work in progress](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=5087%3A87612)

## Tracking

[LET-1211](https://vizzuality.atlassian.net/browse/LET-1211)
